.travis.yml

```diff
@@ -10,8 +10,11 @@ php:
 
 sudo: false
 
+before_install:
+  - composer self-update
```

Comment:
There is no need for this anymore as [travis already do it here](https://travis-ci.org/reactphp/socket/jobs/190057478#L146) while your update call runs [here](https://travis-ci.org/reactphp/socket/jobs/190057478#L161).